src/pudl/transform/eia861.py

```diff
@@ -1020,6 +1014,86 @@ def _harvest_associations(dfs: list[pd.DataFrame], cols: list[str]) -> pd.DataFr
     return assn
 
 
+def _combine_88888_values(df: pd.DataFrame, idx_cols: list[str]) -> pd.DataFrame:
+    """Combine 88888 utility_id_values.
+
+    This function sums rows with a utility_id_eia of 88888 into a single row by
+    primary key. It drops rows with a utility_id_eia of 88888 if there are non-numeric
+    columns with different values that are impossible to combine. E.g.: boolean
+    columns where one value is Y and the other is N. This results in a small loss of
```

Comment:
non-blocking: you mention a few times that the loss of data is small - might be good to quantify that in the docs so people know what to expect.